### PR TITLE
Fixes on slurm partition config and adds first version of config for Bartol

### PR DESCRIPTION
**Pull request by javierggt**
Fixed parition typo. Also, parition was a key of self.config['Cluster'] and not of self.config. Make the partition parameter optional.

### CODE REVIEW

File: submit.py [hunk 1]
```diff
@@ -292,7 +290,8 @@ def write_general_header(self, f, mem=3000, walltime_hours=14,
         self.write_option(f, '--mem=%d'%(mem*1.1))
         if num_gpus:
             self.write_option(f, "--gres=gpu:%d"%num_gpus)
-        self.write_option(f, "--partition=%s" % self.config["parition"])
+        if "partition" in self.config['Cluster'].keys():
```
Comment (dsschult) [inline submit.py]: Should probably be: 
`if "partition" in self.config['Cluster']:`

No need to use `.keys()`

**Comment by briedel** [comment]
Are there SLURM clusters without partitions? If there aren't, a check that the partition is in the cluster config is a better solution.

**Comment by javierggt** [comment]
Our cluster has partitions, but if the partition is not specified the scheduler decides where to send the job.

File: submit.py (hunk 1 above)
Comment (javierggt) [inline submit.py]: ok, should I fix it and submit a fixup?

Comment (dsschult) [inline submit.py]: Fix it and push to your branch.  It should appear here.

Comment (javierggt) [inline submit.py]: done

Comment (dsschult) [inline submit.py]: Not seeing it.  Did you forget to push the local commit to github?

Comment (briedel) [inline submit.py]: It is at the bottom of the page ;)

**Pull request merged**